src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -18,13 +19,15 @@ public abstract class ScalarGraphType : GraphType
         /// Since GraphQL specifies no response format, Serialize is not
         /// responsible for preparing the scalar for transport to the client. It is only responsible
         /// for generating an object which can eventually be serialized by some transport-focused API.
+        /// <br/><br/>
+        /// This method must handle a value of <see langword="null"/>.
         /// </summary>
-        /// <param name="value">Resolved value. Must not be <see langword="null"/>.</param>
+        /// <param name="value">Resolved value. May be <see langword="null"/>.</param>
         /// <returns>
         /// The returned value of a the result coercion is part of the overall execution result.
         /// Normally this value is a primitive value like String or Integer to make it easy for
         /// the serialization layer. For complex types like a Date or Money Scalar this involves
-        /// formatting the value. Returning <see langword="null"/> indicates a failed conversion.
+        /// formatting the value. Returning <see langword="null"/> is valid for nullable types.
```

Comment:
> is valid for **nullable** types

What do you mean? Nullable CLR types like `string` ? Now I see that each numeric scalar throws from `ParseValue` for `null`. OK. `String` scalar does not throw. OK. But `Guid` scalar ( not nullable CLR type) again does not throw. 😕 ?

Reply:
What about Guid?

Reply:
`GuidGraphType.ParseValue(null)` should (and does) return `null`.

Reply:
`GuidGraphType.ParseLiteral(new NullValue())` should (and does) return `null`.
`GuidGraphType.Serialize(null)` should (and does) return `null`.

Reply:
> Now I see that each numeric scalar throws from ParseValue for null

Reply:
Yes.

Reply:
> Suggestions to fix this xml comment would be appreciated.

Already done?

Reply:
No I have not changed it yet.